ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/BlobSidecarValidator.java

```diff
@@ -189,22 +188,22 @@ current shuffling (defined by block_parent_root/slot). If the proposer_index can
             });
   }
 
-  private boolean isSignatureValidWithRespectToProposerIndex(
-      SignedBlobSidecar signedBlobSidecar, BeaconState postState) {
+  private boolean verifyBlobSidecarSignature(
+      BeaconState state, SignedBlobSidecar signedBlobSidecar) {
```

Comment:
i might just tweak these to be final before merge, but looks good...

Reply:
Good call. Thanks! 👍